platform/php/MailChecker.php

```diff
@@ -1,16 +1,7 @@
 <?php
-namespace Fgribreau\PHP;
 
-/**
- * MailChecker
- *
- * Usage:
- *
- * include('mailchecker/platform/php/Mailchecker.php');
- * MailChecker::isValid(string email): bool;
- *
- * @return {Boolean} true is the specified email is valid, false otherwise
- */
+namespace Fgribreau;
```

Comment:
This is the main error. 
In composer, we said `Fgribreau` namespace is in `platform/php`. But this was wrong. It was `Fgribreau\PHP`. So the class was not autoloadable.

Two way to fix it
1. fix the namespace here, to remove PHP. I prefer this one. `PHP` in a *PHP* namespace brings nothing
1. fix in composer json, to append `\\PHP`

As you can see, I choose. This is a BC Break. **BUT** since it does not work anyway, this is a bugfix

---

BTW, the [README](https://github.com/FGRibreau/mailchecker#from-3x-to-4x)  was already OK